modules/chaostesting/src/main/java/org/testcontainers/client/commandparts/SupportedTimeUnit.java

```diff
@@ -0,0 +1,52 @@
+package org.testcontainers.client.commandparts;
+
+/**
+ * Created by novy on 01.01.17.
+ */
+public enum SupportedTimeUnit {
```

Comment:
Do we really have to be that strict? WDYT about using Java's `Duration` and throwing error at runtime instead?

Reply:
Yup, will get rid of it in favour of plain old Duration